bot/cogs/information.py

```diff
@@ -184,6 +184,18 @@ async def server_info(self, ctx: Context) -> None:
 
         await ctx.send(embed=embed)
 
+    @staticmethod
+    def status_to_emoji(status: Status) -> str:
+        """Convert a Discord status into the relevant emoji."""
+        if status is Status.offline:
+            return constants.Emojis.status_offline
+        elif status is Status.dnd:
+            return constants.Emojis.status_dnd
+        elif status is Status.idle:
+            return constants.Emojis.status_idle
+        else:
+            return constants.Emojis.status_online
```

Comment:
Have you considered using a dictionary to define this mapping? This is probably fine, but it feels a little unnatural to me.